src/sentry/static/sentry/app/components/badge.tsx

```diff
@@ -1,16 +1,41 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import {cx} from 'react-emotion';
+import styled from 'react-emotion';
+import space from 'app/styles/space';
+import theme from 'app/utils/theme';
 
-type Props = {
+type Props = React.HTMLProps<HTMLSpanElement> & {
   text?: string;
   priority?: 'strong' | 'new' | 'highlight';
   className?: string;
 };
 
-const Badge = ({priority, className, text}: Props) => (
-  <span className={cx('badge', priority, className)}>{text}</span>
```

Comment:
do we have any tests that rely on `badge` as a className?

Reply:
Not as far as I can tell, they all seem to be passing anyway